.github/workflows/build-refactor-assets-on-release.yml

```diff
@@ -0,0 +1,61 @@
+name: Build assets on release and create archive
+on: 
+  push:
+    tags:
+      - '*.*.*-alpha'
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+          node: [14]
+
+    steps:
+    - name: 'Checkout refactor branch'
+      uses: actions/checkout@refactor
+
+    - name: PHP setup
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: 7.4
+
+    # Install dependencies and handle caching in one go.
+    # @link https://github.com/marketplace/actions/install-composer-dependencies
+    - name: Install Composer dependencies
+      uses: ramsey/composer-install@v1
+
+    - name: Set Node.js version
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node }}
+
+    - name: Cache node modules
+      uses: actions/cache@v1
+      with:
+        path: node_modules
+        key: ${{ runner.OS }}-build-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-build-${{ env.cache-name }}-
+          ${{ runner.OS }}-build-
+          ${{ runner.OS }}-
+
+    - name: Install node packages
+      run: npm install
+
+    - name: Install node packages
+      run: npm run build --no-dev --no-progress
+
+    - name: Archive Release
+      uses: thedoctor0/zip-release@master
+      with:
+        filename: 'release.zip'
+        exclusions: '*.git* /*node_modules/* .editorconfig'
+
+    - name: Add artifacts to release
+      uses: softprops/action-gh-release@v1
+      with:
+        files: release-alpha.zip
```

Comment:
If I'm not mistaken this will make your plugin be named `release-alpha` which is not something you want, right?

Reply:
Yup, I'll correct it